common/io/src/kseqpp_fasta_parser.cpp

```diff
@@ -28,16 +30,34 @@ FastaParserKseqpp::FastaParserKseqpp(const std::string& fasta_file, int min_sequ
 {
     klibpp::KSeq record;
     klibpp::SeqStreamIn iss(fasta_file.data());
+
+    /*    std::ifstream query_file(fasta_file);
```

Comment:
@rached-ab let's remove the commented out code

Reply:
@tijyojwad do you think it is needed to add try/catch in non regression tests when parser in created ?